### DISCUSSION TITLE
Visual Studio code query

### DISCUSSION THREAD

**Discussion by dbeaver12** [microsoft/vscode-discussions#666] [Extension Show and Tell]
When was the software last updated?
•	Is the software still in active development?
•	Are the software provider committed to patching the software in the event of a security incident or known security vulnerabilities?

**Comment by gjsjohnmurray** [comment]
You previously asked this at #665 and received a response there.

**Reply by amandaabas1990** [reply]
#665

**Comment by dbeaver12** [comment]
Hi,

We haven’t yet received a response for the below queries in Visual Code.-
1. When was software last updated
2. Is software still on active development
3. Are the software provider committed to patching the software in event of security incident or known security vulnerabilities?
We would also like to know a list of queries for our Info sec team regarding Visual Studio Code..

Thanks,

Jesu Mathew
Software Licensing Administrator
[https://www.library.mmu.ac.uk/images/mmu_logo.png]

**Reply by gjsjohnmurray** [reply]
Why are you asking these questions on a forum intended for developers who are creating extensions for VS Code?

Please consult your manager for guidance, as your actions risk damaging the professional reputation of the institution you claim to work for.
